src/page.h

```diff
@@ -486,6 +488,9 @@ class page
     index2(uint64_t page_no = 0, bool refresh_page = false)
     {
 
+        // Use commas for thousands separator
+        setlocale(LC_NUMERIC, "en_US.UTF-8");
```

Comment:
As far as I know, this doesn't negatively affect anything. I have not seen any issues with it on my test server.